os/net/bluetooth/bt_conn.c

```diff
@@ -599,7 +599,7 @@ FAR struct bt_conn_s *bt_conn_lookup_handle(uint16_t handle)
 }
 
 /****************************************************************************
- * Name: bt_conn_lookup_addr_le
+ * Name: bt_conn_lookup_addrs_le
```

Comment:
How about modifying the function name with postfix '_internal' for all of duplicated functions.
- bt_conn_lookup_addr_le_internal